current_benchmark.txt

```diff
@@ -0,0 +1 @@
+dropwizard
```

Comment:
Please remove this file. In fact, this file SHOULD be in `.gitignore`, but I will make that change elsewhere.